### PR TITLE
Added columns in document table without testing

### LINKED ISSUE #11: [Sub-Feature] Update SQL queries to extract ratings from columns and populate objects
**Description:**
After the columns in the Documents table have been updated, we also need to update the corresponding SQL queries that allows the middleware to extract ratings from columns and populate objects. 

**Acceptance Criteria:**
The project should still be able to build and add unit tests if necessary. 

**Dependencies:**
This should be completed after the database successfully updates. 

**Estimated Effort:**
If nothing goes wrong, this should be completed in < 1 day.

### PR DESCRIPTION
**Pull request by UncIeRick**
Resolves #11 
**Description**
Added 4 columns to the document table:

- techAvg
- fitAvg
- interpersonalAvg
- numReviews

These columns allow us to keep track of a running average for each rating category, as well as the percentage of users who have rated the document.

The database version has been updated to include the most up-to-date changes, including changes to the database model, DAO, DTO, and SQL queries. These changes are reflected in DocumentDao.java and DocumentDto.java.

**Testing**
Unit tests will be written in the next PR [#13].

### CODE REVIEW

File: docs-core/src/main/resources/db/update/dbupdate-028-0.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+alter table T_DOCUMENT add column DOC_AVG_TECH_C varchar(100);
```
Comment (jakezych) [inline docs-core/src/main/resources/db/update/dbupdate-028-0.sql:1]: What's the reason for the rating columns being strings and not ints?

File: docs-core/src/main/java/com/sismics/docs/core/model/jpa/Document.java [hunk 2]
```diff
@@ -118,7 +118,51 @@ public class Document implements Loggable {
      */
     @Column(name = "DOC_DELETEDATE_D")
     private Date deleteDate;
+
+    @Column(name = "DOC_AVG_TECH_C")
+    private String avg_tech;
+
+    public String getAvg_tech() {
+        return avg_tech;
```
Comment (jakezych) [inline docs-core/src/main/java/com/sismics/docs/core/model/jpa/Document.java:126]: I think for consistency with the rest of the codebase, we should use camel case across all of the changes we make

**Review by jakezych — COMMENTED** [review]
Minor style comments

File: docs-core/src/main/resources/db/update/dbupdate-028-0.sql (hunk 1 above)
Comment (awutis) [reply docs-core/src/main/resources/db/update/dbupdate-028-0.sql:1]: We were thinking what if the average could be a float, so we stored it as a string instead (to allow for both int and float values to be stored, if needed). If we are sure that the average will be either an int or a float, we can change the type

File: docs-core/src/main/java/com/sismics/docs/core/model/jpa/Document.java (hunk 2 above)
Comment (UncIeRick) [reply docs-core/src/main/java/com/sismics/docs/core/model/jpa/Document.java:126]: I've updated the code style with camel cases

File: docs-core/src/main/resources/db/update/dbupdate-028-0.sql (hunk 1 above)
Comment (jakezych) [reply docs-core/src/main/resources/db/update/dbupdate-028-0.sql:1]: Ah right I didn't think about decimals. Is there not a way to store them just as floats? We can still represent ints using floats (4 -> 4.0)

**Comment by jakezych** [comment]
Gonna take the column type discussion offiline. PR compiles but tests fail. Annie's PR will resolve this so gonna go ahead and merge both.

**Pull request merged**